src/centralserver/admin-service/src/main/resources/openapi-definition.yaml

```diff
@@ -1,357 +1,4105 @@
+          format: date-time
+          maxLength: 24
+          minLength: 24
+          type: string
+      required:
+        - code
+        - description
+      type: object
+    GlobalGroupCodeAndDescription:
+      description: group code and description
+      properties:
+        code:
+          description: group code
+          example: groupcode
+          format: text
+          maxLength: 255
+          minLength: 1
+          type: string
+        description:
+          description: group description
+          example: description
+          format: text
+          maxLength: 255
+          minLength: 1
+          type: string
+      required:
+        - code
+        - description
+      type: object
+    GlobalGroupDescription:
+      description: group description
+      properties:
+        description:
+          description: group description
+          example: description
+          format: text
+          maxLength: 255
+          minLength: 1
+          type: string
+      required:
+        - description
+      type: object
+    GroupMember:
+      description: group member
+      properties:
+        created_at:
+          description: group member created at
+          example: '2022-01-20T00:00:00.001Z'
+          format: date-time
+          maxLength: 24
+          minLength: 24
+          type: string
+        id:
+          description: group member id
+          example: FI:GOV:123:SS1
+          format: text
+          maxLength: 1023
+          minLength: 1
+          type: string
+        name:
+          description: group member name
+          example: Member123
+          format: text
+          maxLength: 255
+          minLength: 1
+          type: string
+      required:
+        - id
+        - name
+        - created_at
+      type: object
+    HighAvailabilityStatus:
+      description: Central server's high availability status
+      properties:
+        is_ha_configured:
           default: false
+          description: indicates if the central server is in high availability mode
+          type: boolean
         node_name:
+          description: central server's node name
+          example: node_0
+          format: text
+          maxLength: 255
+          minLength: 1
+          type: string
+      type: object
+    InitialServerConf:
+      allOf:
+        - $ref: '#/components/schemas/InstanceIdentifier'
+        - $ref: '#/components/schemas/CentralServerAddress'
+      description: central server initial configuration
+      properties:
+        software_token_pin:
+          description: pin code for the initial software token
+          example: sup3rs3cr3t_p!n
+          format: text
+          maxLength: 255
+          minLength: 1
+          type: string
+      required:
+        - software_token_pin
+      type: object
+    InitializationStatus:
+      allOf:
+        - $ref: '#/components/schemas/InstanceIdentifier'
+        - $ref: '#/components/schemas/CentralServerAddress'
```

Comment:
This feels somewhat confusing. In SS InitializationStatus is a collection of status booleans, here it is one status enum, and two data properties. 

Having identical name and resource for different types of animals throws me off, it would be better to a) either also have boolean is_xxx_done fields also in CS.InitializationStatus, or b) different name than InitializationStatus.

Reply:
Yeah, being consistent with the Security Server initialization could be something to consider here. 

The initialization part (controller layer, service layer, logic – the works) has been implemented already quite some time ago so I kind of did not want to start an overhaul for that at this point.

Reply:
Ah yes, of course, in that case doesn't make sense to change it in this ticket.